src/Extensions/PageControllerExtension.php

```diff
@@ -4,15 +4,16 @@
 
 use SilverStripe\SiteConfig\SiteConfig;
 use SilverStripe\View\Requirements;
-//todo: add Extension use statement
+use SilverStripe\core\Extension;
 
 class PageControllerExtension extends Extension
 {
+
     public function contentcontrollerInit()
-    {
+    {  
         if (
             ! empty(SiteConfig::current_site_config()->GetSiteControlAPI) &&
-            $this->IsGetSiteControlPage()
+            ! $this->owner->IsGetSiteControlPage()
```

Comment:
why the NOT (!) operator?

Reply:
That stilll does not make any sense.

`IsGetSiteControlEnabledOnPageLevel()` is a method that returns true if the code should be included:

![image](https://user-images.githubusercontent.com/167154/153275409-f4beab63-4490-47cc-8aad-e69d6d872707.png)

Please reconsider.